tests/integration/test_charm_azure.py

```diff
@@ -92,10 +91,10 @@ async def test_build_and_deploy(ops_test: OpsTest, charm_versions, azure_credent
     delete_azure_container(azure_credentials["container"])
     sleep(10)
     # Setup container
-    setup_azure_container_for_history_server(
-        azure_credentials["container"], azure_credentials["path"]
-    )
-    logger.info("Azure container and path correctly setup!")
+    # setup_azure_container_for_history_server(
+    #     azure_credentials["container"], azure_credentials["path"]
+    # )
+    # logger.info("Azure container and path correctly setup!")
```

Comment:
TODO: Remove this commented code as well as the helpers once we confirmed green CI